code/modules/mob/living/carbon/alien/humanoid/queen.dm

```diff
@@ -66,6 +68,22 @@
 	internal_organs += new /obj/item/organ/alien/eggsac
 	..()
 
+/mob/living/carbon/alien/humanoid/royal/queen/proc/game_end()
+	if(stat != DEAD)
+		SSshuttle.clearHostileEnvironment(src)
+		if(EMERGENCY_IDLE_OR_RECALLED)
+			priority_announce("Xenomorph infestation detected: crisis shuttle protocols activated - jamming recall signals across all frequencies. P.S.: Bring back a live one please.")
+			SSshuttle.emergency.request(null, set_coefficient=0.5)
+			SSshuttle.emergencyNoRecall = TRUE
+
+/mob/living/carbon/alien/humanoid/royal/queen/death() //dead queen doesnt stop shuttle
+	SSshuttle.clearHostileEnvironment(src)
+	..()
+
+/mob/living/carbon/alien/humanoid/royal/queen/Destroy()
+	SSshuttle.clearHostileEnvironment(src)
+	..()
```

Comment:
I wonder if there should be an announcement for this...

Reply:
> I wonder if there should be an announcement for this...

Probably not, as by that time everyone is most likely dead.

Reply:
There should be